CODEOWNERS

```diff
@@ -149,6 +149,7 @@
 /bundles/org.openhab.binding.hpprinter/ @cossey
 /bundles/org.openhab.binding.http/ @openhab/add-ons-maintainers
 /bundles/org.openhab.binding.hue/ @cweitkamp @andrewfg
+/bundles/org.openhab.binding.huesync/ @openhab/add-ons-maintainers
```

Comment:
@pgfeller this should be your GIT id here

Reply:
In the first version I had my id there, but as I'm not fluent in Java (I only use it for the projects here), nor have expertise in regards to the openHAB stack and their respective best practices I do not feel qualified to do a competent job as a maintainer (e.g. code review if someone else creates a PR for this binding).

I once was maintainer of the iCloud binding and realized that due to time/resource limitations on my side, the diversity of the devices out there and the problems introduced by using an unofficial API I could not do the job properly (since I do not use apple devices anymore I'm not the maintainer anymore for this binding).

That said: Here we have a defined API and a single product - I do not expect this amount of troubles. As long as I have a bridge in use I'll the binding for my own setup and will keep it functional ... 

➡️ I considered myself more as a contributor - but I also would like to lessen the burden on the _add-ons-maintainers_. Therefore - If despite the limitations you think I can be of help as a maintainer I'll do my best to keep this thing flying as long as I have the hardware in use; but would seek advice in case I face something I feel I need a more educated/experienced person to judge a change.

Reply:
I would say that you are the most expert person in the community concerning this device.